package.json

```diff
@@ -37,6 +37,7 @@
     "debug": "^4.1.1",
     "dotenv": "^8.0.0",
     "express": "^4.17.1",
+    "faker": "^4.1.0",
```

Comment:
Since this is not used in the main package, I suggest you include it in the devDependencies.